templates/search_page.html

```diff
@@ -136,9 +136,22 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                     <div class="flex-fill">
                         <input class="form-control border-0" type="search" id="location" placeholder="Search"
-                            aria-label="Search" name="location" value="Los Angeles, CA, USA">
+                            aria-label="Search" name="location" value="" onchange="fetchLoc(this)">
+                        <script>
+                            try {
```

Comment:
can we try moving these JS logics to `search.js`? usually long JS logics (more than a couple of lines) are better to place in one file for better readability.

Reply:
Some of the stuff was moved , some cannot be